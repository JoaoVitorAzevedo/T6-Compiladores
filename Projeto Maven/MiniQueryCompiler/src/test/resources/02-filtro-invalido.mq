LOAD "data.json"
FILTER .nome > 18  # Erro: string com operador num√©rico
SELECT .email