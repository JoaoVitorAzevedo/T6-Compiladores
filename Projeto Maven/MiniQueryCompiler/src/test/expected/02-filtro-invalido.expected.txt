✖ ERROS SEMÂNTICOS:
ERRO (Linha 2): Tipo incompatível: o campo '.nome' é do tipo string, mas o valor '18' é do tipo number
ERRO (Linha 2): Operador '>' inválido para strings. Use '==' ou '!='.